Realm/Realm.UnityWeaver/UnityWeaver.cs

```diff
@@ -120,7 +121,7 @@ public static void Initialize()
 
                 WeaveAssemblyCore(assemblyPath, assembly.allReferences,
                     Metric.Framework.UnityEditor, Application.unityVersion,
-                    GetTargetOSName(Application.platform), netFrameworkInfo.TargetArchitecture,
+                    GetTargetOSName(Application.platform), GetHostCpuArchitecture(),
```

Comment:
With this you want to say that the target arch is the same as the host's arch as it's mono building for the editor which is effectively running on the host/dev's machine?
Same question for the same change below.

Reply:
Yes, since the editor build is going to be deployed on the same machine it's running on, it's safe to assume the editor architecture is the same as the app architecture.